chain/params_test.go

```diff
@@ -54,3 +56,59 @@ func TestParamsForksInTime(t *testing.T) {
 	expect("constantinople", ff.Constantinople, false)
 	expect("eip150", ff.EIP150, false)
 }
+
+func TestParams_CalculateBurntContract(t *testing.T) {
+	tests := []struct {
```

Comment:
Introduce parallel execution?
```suggestion
        t.Parallel()
	tests := []struct {
```

Reply:
Done